### PR TITLE
Side by side form summary views

### PR DESCRIPTION
**Pull request by proteusvacuum**
This is the beginning of having side by side diff views in the app summary.
This is currently hidden as there are no links to this page - I will add links once we are ready to release it.

Still to come is something similar for the case summary, and an option to show the difference between the two versions.

Screenshot from 2019-04-05 15-41-05

FF: not really a FF, but hidden at the url `{domain}/apps/compare/{app_id_1}..{app_id_2}`

### CODE REVIEW

**Comment by orangejenny** [comment]
Ooooooh, this is exciting. Two ui thoughts/questions:
- How well does this degrade as the window shrinks? I think it's okay that this is inherently a wide layout, just curious. Might be worth considering implementing the collapsing side menu that reports uses.
- Have you considered / would you consider scrolling the two versions independently?

**Comment by orangejenny** [comment]
> Have you considered / would you consider scrolling the two versions independently?

*cough* So now that I've actually looked at the commit list, I see you did this. Nevermind.

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/app_manager/templates/app_manager/form_summary_diff.html [hunk 1]
```diff
@@ -6,17 +6,25 @@
 
 {% requirejs_main "app_manager/js/summary/form_diff" %}
 
+{% block stylesheets %}
+    {{ block.super }}
+    <link type="text/css"
+          rel="stylesheet"
+          media="all"
+          href="{% static 'app_manager/css/app_summary_diff.css' %}" />
```
[the remark this answers is not in the document]
Comment (proteusvacuum) [reply corehq/apps/app_manager/templates/app_manager/form_summary_diff.html]: Oops, I didn't see that file existed. Updated

**Comment by proteusvacuum** [comment]
@orangejenny re: page degradation as the window shrinks: it's not amazing, as the right pane ends up beneath the first one, making this comparison view not very useful. This happens even if I hide the sidebar. I think if we want to make this "mobile accessible", we might need to do another design pass at it.

**Comment by orangejenny** [comment]
It's too bad hiding the sidebar didn't help. Seems fine to deal with this in a later iteration (if ever).

**Comment by orangejenny** [comment]
You've still got the link tag referencing `app_summary_diff.css`

**Comment by proteusvacuum** [comment]
Oops, will fix in a followup. Thanks.

**Pull request merged**